src/sonic-yang-models/yang-models/sonic-static-route.yang

```diff
@@ -0,0 +1,124 @@
+module sonic-static-route {
+  yang-version 1.1;
+  namespace "http://github.com/Azure/sonic-static-route";
+  prefix sroute;
+
+  import sonic-vrf {
+    prefix vrf;
+  }
+  import ietf-inet-types {
+    prefix inet;
+  }
+  import sonic-types {
+    prefix stypes;
+  }
+
+  organization
+    "SONiC";
+  contact
+    "SONiC";
+  description
+    "STATIC ROUTE yang Module for SONiC OS";
+
+  revision 2022-03-17 {
+    description
+      "First Revision";
+  }
+
+  container sonic-static-route {
+    container STATIC_ROUTE {
+      description
+        "STATIC_ROUTE part of config_db.json";	
+	list STATIC_ROUTE_TEMPLATE_LIST {
+        key "prefix";
+        leaf prefix {
+          type inet:ip-prefix;
+          description
+            "prefix is the destination IP address, as key";
+        }
+        leaf nexthop {
+          type string;
+          description
+            "The next-hop that is to be used for the
+             static route as IP address. When interface needs to be
+             specified, use 0.0.0.0 as leaf value";
+        }
+        leaf ifname {
+          type string;
+          description
+            "When interface is specified, forwarding happens through it";
+        }
+        leaf advertise {
+          type string{
+            pattern "((true|false),)*(true|false)";
+                  }
+          default "false";

```

Comment:
can you add unit test for this field?

Reply:
Unit test has been added for STATIC_ROUTE_LIST.  STATIC_ROUTE_TEMPLATE_LIST was added to help with bgp unit tests. The STATIC_ROUTE schema in the community has only the prefix (no vrf -name as key).